modules/querying-the-status-of-cluster-components-using-the-cli.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="querying-the-status-of-cluster-components-using-the-cli_{context}"]
+= Querying the status of cluster components by using the CLI
+
+You can verify the status of certificate signing requests (CSRs), cluster Operators, nodes, and the pods running in core {product-title} projects.
```

Comment:
I am surprised that we are directing users to do these things as part of validating an installation.
1) I would not expect the user to have to look at CSRs at all unless the user is trying to diagnose why the cluster does not have the nodes that the user expects.
2) I would not expect the user to look at ClusterOperator resources directly. I would expect the user to look at the aggregated status in the ClusterVersion resource to ascertain whether there are cluster operators that are not available.
3) I would not expect the user to look at pods unless the user is trying to diagnose why a specific ClusterOperator is not available.

In other words, as part of validating an installation, I would only expect the user to look at ClusterVersion and Nodes. If either of those are showing problems, then looking at things like CSRs, ClusterOperators, and Pods may be something that they would then do, but that would be part of troubleshooting rather than validating.

Reply:
@staebler thank you for your guidance. I have subsequently made the following changes to this PR:

- I have updated the title for this section and the content so that it focuses on the nodes.

- I have updated the [Getting cluster version and update details](https://osdocs-1688-validating-an-installation--ocpdocs.netlify.app/openshift-enterprise/latest/installing/validating-an-installation.html#getting-cluster-version-and-update-details_validating-an-installation) section of this document to include a step stating that if the cluster status indicates that the installation is still progressing, you can obtain more detailed progress information by checking the status of the Operators.

- I have included a new additional resources link after the [Getting cluster version and update details](https://osdocs-1688-validating-an-installation--ocpdocs.netlify.app/openshift-enterprise/latest/installing/validating-an-installation.html#getting-cluster-version-and-update-details_validating-an-installation) section, linking to the [Querying Operator status after installation](https://osdocs-1688-validating-an-installation--ocpdocs.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-installations.html#querying-operator-status-after-installation_troubleshooting-installations) section of the Troubleshooting Guide.

- I have updated the [Querying Operator status after installation](https://osdocs-1688-validating-an-installation--ocpdocs.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-installations.html#querying-operator-status-after-installation_troubleshooting-installations) section of the Troubleshooting Guide to include some details about checking that CSRs are approved.